index.js

```diff
@@ -46,7 +47,7 @@ var gs = {
       stream.write({
         cwd: opt.cwd,
         base: basePath,
-        path: path.resolve(opt.cwd, filename),
+        path: filename,
```

Comment:
oh, an absolute glob has already been resolved to this and the filename is actually the whole path?